Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -57,34 +59,35 @@ namespace gdjs {
       for (let i = 0; i < this._borderSprites.length; ++i) {
         this._spritesContainer.addChild(this._borderSprites[i]);
       }
-      this._alpha = this._spritesContainer.alpha;
+      this._wrapperContainer.addChild(this._spritesContainer);
+      this._alpha = this._wrapperContainer.alpha;
       runtimeScene
         .getLayer('')
         .getRenderer()
-        .addRendererObject(this._spritesContainer, runtimeObject.getZOrder());
+        .addRendererObject(this._wrapperContainer, runtimeObject.getZOrder());
     }
 
     getRendererObject() {
-      return this._spritesContainer;
+      return this._wrapperContainer;
     }
 
     ensureUpToDate() {
       if (this._spritesContainer.visible && this._wasRendered) {
         // Update the alpha of the container to make sure that it's applied.
         // If not done, the alpha will be back to full opaque when changing the color
         // of the object.
-        this._spritesContainer.alpha = this._alpha;
+        this._wrapperContainer.alpha = this._alpha;
         this._spritesContainer.cacheAsBitmap = true;
       }
       this._wasRendered = true;
     }
 
     updateOpacity(): void {
-      //TODO: Workaround a not working property in PIXI.js:
-      this._spritesContainer.alpha = this._spritesContainer.visible
-        ? this._object.opacity / 255
-        : 0;
-      this._alpha = this._spritesContainer.alpha;
+      // The alpha is updated on a wrapper around the sprite because a known bug
+      // in Pixi will create a flicker when cacheAsBitmap is set to true
+      // (see https://github.com/pixijs/pixijs/issues/4610)
+      this._wrapperContainer.alpha = this._object.opacity / 255;
+      this._alpha = this._wrapperContainer.alpha;
```

Comment:
This might be a good opportunity to scrap entirely this `_alpha` property. I *think* it's now useless, because we update directly the alpha on the wrapper container - no need to store it to apply it later. See my other comment in `ensureUpToDate`.